public/javascripts/app.js

```diff
@@ -0,0 +1,19 @@
+import io from 'socket.io-client';
```

Comment:
This file should not be available to browser through `/javascripts/app.js`

Reply:
A client library that loads on the browser side: socket.io-client 
How can I use socket client other way??

Reply:
As discussed in private messaging, this comment is about `this file`, not about socket.io